src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -11,7 +11,7 @@
 
   // rate of failing to fire torpedos [0.0, 1.0]
   private double FAILURE_RATE = 0.0; //NOSONAR
-
+  private Random rand = new SecureRandom();
```

Comment:
Please change name of variable rand to generator as it was previously named as generator.